### ISSUE TITLE
Feature request: merge tag log

### ISSUE THREAD

**Issue by siddharthvp** [azatoth/twinkle#879]
From [WT:TW](https://en.wikipedia.org/wiki/Wikipedia_talk:Twinkle#Feature_request:_merge_log)

> Per Wikipedia:Village_pump_(technical)#Any_tool_that_can_create_a_log_of_mergers_I_proposed?. I think it would be highly useful and easy to implement, and complementary to CSD and PROD log features.  -- Piotrus 15:23 pm, 28 February 2020 (UTC) 

Adding still more complexity to the tag module not withstanding, I seem to agree that this would be useful. Merge discussions are a slow process - often there are no comments months after being opened, that the person who started the discussion himself would forget about it after some time. Having a merge tag log makes it easier to track down the fate of these discussions. 

This FR inspired me to do #878, which reduces the additions needed in friendlytag.

**Comment by DannyS712** [comment]
I was literally going to propose the same thing that you started in #878 - especially if logging is going to be expanded. I suggest waiting until that is resolved before expanding logging

**Comment by Amorymeltzer** [comment]
Do we have a good way to quantify how many such events occur, and via Twinkle?  I guess it'd be useful but it'd be good to get a sense of how niche it'd be.

My main worry is proliferation of logs.  Prod and CSD make sense, XfD (#839) is a big collection like CSD and broadly applicable to most users.   Draftification exists through another tool, but wouldn't have been my choice. Merge could be good but what about PNT?  Hard no on that, should there be an RfPP log?  An AIV/ARV log?  There's value there (SQL had a tool that didn't survive a recent DB change) but should be discussed more broadly.

tl; dr: Slippery slope, blah blah blah

**Comment by Amorymeltzer** [comment]
At any rate, path forward is #838 then go ahead and work on this if you like @Siddharthvp.

**Comment by siddharthvp** [comment]
I guess you're quite right. I already have [a tool](https://draftifyhistory.toolforge.org/) for tracking draftifications, might as well make a similar one to track merge taggings done via TW. This sounds like a better solution.

**Comment by Amorymeltzer** [comment]
Oh nice!  That actually seems like a *perfect* idea.
